Assignment1 Questions

```diff
@@ -0,0 +1,13 @@
+/* Matthew Danielson
+*  6/28/16
+* Part 2: Answers to Questions
+* Note: Sorry this is a touch late, I needed some help wthi uploading to github
+*/
+
+1) There are no static voids in C, and any method you run needs to have a return statement.  Additionally, it denotes that there are no errors for the computer to take note of.
+
+2) No return statement; a integer method is being run, thus an integer must be returned. While not strictly necessary, not including this can cause erros.
```

Comment:
Correct, but you're missing a couple of errors - see if you can find the other errors!